### DISCUSSION TITLE
Connec to MQTT Broker via mqtt protocol

### DISCUSSION THREAD

**Discussion by yellsri-bh** [mqttjs/MQTT.js#1949] [General]
I have broker with mqtt and ws protocol
1. Tried to connect to broker via mqtt protocol, its not getting connected but working with ws protocol
2. If mqtt:// is specified in the brokerUrl, internally replacing to ws://.

Does this package support to connect to broker via mqtt protocol ?

**Comment by robertsLando** [comment]
Answered in the issue, browser only support ws/wss protocol. You can use all other protocols with nodejs
